pkg/cmd/pr/checkout/checkout_test.go

```diff
@@ -426,26 +345,18 @@ func TestPRCheckout_differentRepo_existingBranch(t *testing.T) {
 	} } } }
 	`))
 
-	ranCommands := [][]string{}
-	//nolint:staticcheck // SA1019 TODO: rewrite to use run.Stub
-	restoreCmd := run.SetPrepareCmd(func(cmd *exec.Cmd) run.Runnable {
-		switch strings.Join(cmd.Args, " ") {
-		case "git config branch.feature.merge":
-			return &test.OutputStub{Out: []byte("refs/heads/feature\n")}
-		default:
-			ranCommands = append(ranCommands, cmd.Args)
-			return &test.OutputStub{}
-		}
-	})
-	defer restoreCmd()
+	cs, cmdTeardown := run.Stub()
+	defer cmdTeardown(t)
+
+	cs.Register(`git fetch origin refs/pull/123/head:feature`, 0, "")
+	cs.Register(`git config branch\.feature\.merge`, 0, "")
```

Comment:
This stub used to print the following contents as output: `refs/heads/feature\n`